docs/status_screen.txt

```diff
@@ -397,6 +397,9 @@ directory. This includes:
   - variable_paths - number of test cases showing variable behavior
   - unique_crashes - number of unique crashes recorded
   - unique_hangs   - number of unique hangs encountered
+  - command_line   - full command line used for the fuzzing session
+  - slowest_exec_ms- real time of the slowest execution in ms
```

Comment:
> jonathanmetzman:
Are we naming/making this `slowest_exec_ms` instead of libFuzzer's `slowest_unit_time_sec` because the other AFL time stats are reported in ms and because AFL's timeout is usually < 1 second?
\
Maybe this should be named slowest_exec to be consistent with exec_timeout?

  
 > neuracr
I know that libFuzzer measures slow units in seconds. But I think that since we gather the times in ms granularity (even μs) it's better to keep ms, because seconds seem a bit too coarse in a fuzzing context.
Talking about the name, if we decide to keep it in ms, I think it's better to mention it so that people don't get confused. I don't have an opinion for `unit` or `exec`.

Reply:
ACK. I don't know if we'll grab this stat in ClusterFuzz since we stopped surfacing the libFuzzer version a while ago. But if we did grab it, it would be much nicer if the stat were named the same and had the same units.